src/playground/src/ui/screen/topNavigationBar.component.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  Alert,
+} from 'react-native';
+import {
+  NavigationScreenProps,
+  SafeAreaView,
+} from 'react-navigation';
+import {
+  withStyles,
+  ThemeType,
+  ThemedComponentProps,
+} from '@kitten/theme';
+import {
+  TopNavigationBar as TopNavigationBarComponent,
+  TopNavigationBarAction,
+} from '@kitten/ui';
+
+type Props = & ThemedComponentProps & NavigationScreenProps;
+
+const leftControlUri: string = 'https://pngimage.net/wp-content/uploads/2018/05/back-icon-png-6.png';
+const rightControlUri1: string =
+  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQLai-QccSTB98vmP7dUigtL8F13cE6_0mmju_1m4mDQgzLd_Zq';
+const rightControlUri2: string =
+  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQt6ezD4tiscqX5g4AFKa0vFC0z7rLTlAJn53Jl65uZ5JfPq4Ot';
+
+class TopNavigationBar extends React.Component<Props> {
+
+  static navigationOptions = {
+    header: (props: NavigationScreenProps) => (
```

Comment:
I think it would be nice if we can provide user-friendly interface for this
Like HOCs(?) which would automatically fill-in header property with provided Header Component